script/foundry/deployment/Main.s.sol

```diff
@@ -27,20 +27,10 @@ contract Main is DeployHelper {
     /// forge script script/foundry/deployment/Main.s.sol:Main --rpc-url $RPC_URL --broadcast --verify -vvvv
 
     function run() public virtual override {
-        bool configByMultisig;
-        try vm.envBool("DEPLOYMENT_CONFIG_BY_MULTISIG") returns (bool mult) {
-            configByMultisig = mult;
-        } catch {
-            configByMultisig = false;
-        }
-        console2.log("configByMultisig:", configByMultisig);
-
         // deploy all contracts via DeployHelper
         super.run(
```

Comment:
No longer need to have an option to config using multisig?

Reply:
It's not very practical for deployments. We are going to use a fresh, safe EOA then grant roles to the multisig and revoke all roles of the deployer at the end. That's why we introduce checks to make sure.
Of course after a deployment we need to smoke test admin roles for correctness.